go.mod

```diff
@@ -0,0 +1,128 @@
+module github.com/rancher/k3s
+
+go 1.12
+
+replace (
+	github.com/containerd/containerd => github.com/rancher/containerd v1.2.8-k3s.1
+	github.com/containerd/cri => github.com/rancher/cri v1.2.8-k3s.1
+	github.com/containernetworking/plugins => github.com/rancher/plugins v0.7.5-k3s1
+	github.com/coreos/flannel => github.com/ibuildthecloud/flannel v0.10.1-0.20190131215433-823afe66b226
+	github.com/kubernetes-sigs/cri-tools => github.com/rancher/cri-tools v1.15.0-k3s.2
+	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
+	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v0.0.0-20180911193056-5684b8af48c1
+	github.com/rancher/dynamiclistener => github.com/erikwilson/rancher-dynamiclistener v0.0.0-20190717164634-c08b499d1719
+	github.com/rancher/kine => github.com/ibuildthecloud/kine v0.1.0
+	golang.org/x/sys => github.com/golang/sys v0.0.0-20190204203706-41f3e6584952
+	k8s.io/api => ./kubernetes/staging/src/k8s.io/api
+	k8s.io/apiextensions-apiserver => ./kubernetes/staging/src/k8s.io/apiextensions-apiserver
+	k8s.io/apimachinery => ./kubernetes/staging/src/k8s.io/apimachinery
+	k8s.io/apiserver => ./kubernetes/staging/src/k8s.io/apiserver
```

Comment:
looking at https://github.com/golang/go/wiki/modules#publishing-a-release ,
i guess ```k8s.io/apiserver => github.com/rancher/k3s/staging/src/k8s.io/apiserver v1.15.3-k3s.1``` works if you add tags like ```staging/src/k8s.io/apiserver/v1.15.3-k3s.1```

Reply:
Thanks! Let me try this.

Reply:
@yamt Thanks so much, this worked. I can drop the submodule now.